checker/src/org/checkerframework/checker/lock/LockTransfer.java

```diff
@@ -40,12 +46,24 @@
     protected LockChecker checker;
 
     /** Annotations of the lock type system. */
-    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD;
+    protected final AnnotationMirror GUARDEDBY, GUARDSATISFIED, GUARDEDBYUNKNOWN, GUARDEDBYBOTTOM, LOCKHELD, LOCKPOSSIBLYHELD;
 
     public LockTransfer(LockAnalysis analysis, LockChecker checker) {
-        super(analysis);
+        // Always run the Lock Checker with -AconcurrentSemantics turned on.
+        super(analysis, true /* useConcurrentSemantics */);
+        // This assignment is needed (even though the super constructor is called) because
+        // LockTransfer.analysis shadows CFAbstractTransfer.analysis,
         this.analysis = analysis;
         this.checker = checker;
+
+        GUARDEDBY = AnnotationUtils.fromClass(analysis.getTypeFactory()
```

Comment:
Instead of instantiating the AnnotationMirrors again, couldn't you just write `checker.GUARDEDBY`, etc.?